high_card.rb

```diff
@@ -1,5 +1,6 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
-
+ranks = %w{"A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" }
```

Comment:
You only want to use the `%w` method if you're creating an array of strings (i.e., no integers).

I find it simpler to initialize arrays using `[ ]`.